math/minuit2/inc/Minuit2/FCNGradientBase.h

```diff
@@ -41,6 +41,8 @@ class FCNGradientBase : public FCNBase {
    virtual ~FCNGradientBase() {}
 
    virtual std::vector<double> Gradient(const std::vector<double> &) const = 0;
+   virtual std::vector<double> Gradient(const std::vector<double> &parameters, double */*previous_grad*/, double */*previous_g2*/,
+                                        double */*previous_gstep*/) const { return Gradient(parameters); };
```

Comment:
```suggestion
   virtual std::vector<double> GradientWithPrevResult(const std::vector<double> &parameters,
                                                      double * /*previous_grad*/, double * /*previous_g2*/,
                                                      double * /*previous_gstep*/) const { return Gradient(parameters); };
```

Besides the name change, try to not have the character sequence `*/*` but put a space after the first `*` to not confuse syntax highlighting of GitHub and many editors.